packages/react-slider/src/components/Slider/useSliderStyles.ts

```diff
@@ -4,297 +4,135 @@ import type { SliderState } from './Slider.types';
 
 export const sliderClassName = 'fui-Slider';
 
-export const thumbClassName = `${sliderClassName}-thumb`;
-export const trackClassName = `${sliderClassName}-track`;
-export const markContainerClassName = `${sliderClassName}-markItemContainer`;
-export const firstMarkClassName = `${sliderClassName}-firstMark`;
-export const lastMarkClassName = `${sliderClassName}-lastMark`;
-export const markClassName = `${sliderClassName}-mark`;
-export const markLabelClassName = `${sliderClassName}-label`;
+const thumbSizeVar = `--fui-slider-thumb-size`;
+const railSizeVar = `--fui-slider-rail-size`;
+const railColorVar = `--fui-slider-rail-color`;
+const progressColorVar = `--fui-slider-progress-color`;
+const thumbColorVar = `--fui-slider-thumb-color`;
+
+export const railDirectionVar = `--fui-slider-rail-direction`;
+export const railOffsetVar = `--fui-slider-rail-offset`;
+export const railProgressVar = `--fui-slider-rail-progress`;
+export const railStepsPercentVar = `--fui-slider-rail-steps-percent`;
+export const thumbPositionVar = `--fui-slider-thumb-position`;
+
+const get = (value: string) => `var(${value})`;
```

Comment:
I don't know if having a getter such as this is the best thing we can do, I'd personally prefer just using the `var(${value})` directly but I'd be curious as to what other people prefer.

Reply:
yeah, i yanked it out. wasn't as big of a deal/helper as I hoped